src/Components/MenuItems.js

```diff
@@ -5,12 +5,11 @@ import { IoIosArrowUp } from "react-icons/io";
 export const MenuItems = [
     {
         page: 'Home',
-        path: '/Home',
+        path: '/',
         className: 'Nav_link'
     },
     {
         page: 'My Work',
-        path: '/',
```

Comment:
Will removing this “path” attribute here affect where this MenuItems component is being called?